neo/io/nixio.py

```diff
@@ -1167,7 +1167,7 @@ def _write_spiketrain(self, spiketrain, nixblock, nixgroup):
             return
 
         if isinstance(spiketrain, BaseProxy):
-            spiketrain = spiketrain.load()
+            spiketrain = spiketrain.load(load_waveforms=True)
```

Comment:
It would control this in an option at class init, no ?
Otherwise we can swtich to super slow when writing.

Reply:
OK